src/components/Checkout/OrderTotals.vue

```diff
@@ -22,7 +22,7 @@
 			:order-total-variant="donateItemExperimentVersion === 'e'"
 			:loan-count="Number(totals.loanReservationTotal) > 0 ? 1 : 0"
 			:loan-reservation-total="Number(totals.loanReservationTotal)"
-			@refreshtotals="$emit('refreshtotals')"
+			@refresh-totals="$emit('refreshtotals')"
```

Comment:
DonationItem only emits the `refreshtotals` event without the `-` so this will need to be reverted.

Reply:
done